docs/rules.rst

```diff
@@ -4,10 +4,102 @@
 Rules
 *****
 
-.. contents:: Topics
+Specifying Rules at Runtime
+---------------------------
 
-This topic describes how to use the default Ansible Lint rules, as well as how to create and use custom rules.
+By default, ``ansible-lint`` uses the rules found in ``ansible-lint/lib/ansiblelint/rules``. To override this behavior and use a custom set of rules, use the ``-r /path/to/custom-rules`` option to provide a directory path containing the custom rules. For multiple rule sets, pass multiple ``-r`` options.
 
-.. include:: ../README.rst
-   :start-after: rules-docs-inclusion-marker-do-not-remove
-   :end-before: rules-docs-inclusion-marker-end-do-not-remove
+It's also possible to use the default rules, plus custom rules. This can be done by passing the ``-R`` to indicate that the default rules are to be used, along with one or more ``-r`` options.
+
+Using Tags to Include Rules
+```````````````````````````
+
+Each rule has an associated set of one or more tags. To view the list of tags for each available rule, use the ``-T`` option.
+
+The following shows the available tags in an example set of rules, and the rules associated with each tag:
+
+.. code-block:: bash
```

Comment:
"bash" is the wrong lexer choice for this. The correct one is `console` or `shell-session`. Otherwise, it'll attempt to highlight both the prompt and the command output as bash which would look funny.

Ref: https://pygments.org/languages/#interactive-terminal-shell-sessions.